vme/src/namelist.cpp

```diff
@@ -359,22 +360,31 @@ const char *cNamelist::IsNameRaw(const char *name) const
     if (name == nullptr)
         return nullptr;
 
+    int s;
```

Comment:
This should move down to 367 - a good style rule is "declaration is initialization"

Reply:
+1